app/Http/Controllers/Admin/DigitalPublicationArticleController.php

```diff
@@ -44,8 +45,17 @@ protected function indexData($request)
     {
         $digPub = app(DigitalPublicationRepository::class)->getById(request('digitalPublication'));
 
+        $articlesList = $this->repository->withDepth()->defaultOrder()->get()->filter(function ($article) {
+            return $article->depth < 3;
+        })->pluck('title', 'id');
+
+        $articlesList = $articlesList->prepend('None', '');
+
         return [
+            'nested' => true,
+            'nestedDepth' => 3,
```

Comment:
These values are already set in the parent controller, so rather than resetting them here, merge your values with the parent's return value:

```
return array_merge(parent::indexData($request), ['types' => ...]);
```